epub33/core/index.html

```diff
@@ -1411,6 +1411,11 @@ <h4>Resource locations</h4>
 				<p>Storing all resources inside the EPUB container is strongly encouraged whenever possible as it allows
 					users access to the entire presentation regardless of connectivity status.</p>
 
+				<p>When resources have to be located outside the EPUB container, EPUB creators are strongly RECOMMENDED
```

Comment:
Ya, it's usually used for emphasis when we're speaking non-normatively, like the "strongly encouraged" in the paragraph above.

Since this is now a normative statement, "strongly" should definitely be stripped. There's a similar instance on a security statement in the RS spec I'll erase.